website/docs/developer-docs/docs/templates/tutorial-template.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Tutorial Template"
+---
+
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+# Tutorial: [Title of the Tutorial]
+
+## Overview
+
+[A brief introduction explaining what this tutorial will cover and what the user will achieve by following it. Keep this to 2-3 sentences. Include why this task is valuable or important.]
+
```

Comment:
Consider adding a small section for an example use case, maybe as part of the Stated Goals section, to help readers understand why this task is important.